include/class.ticket.php

```diff
@@ -2572,10 +2572,6 @@ static function create($vars, &$errors, $origin, $autorespond=true,
         if (isset($vars['autorespond']))
             $autorespond = $vars['autorespond'];
 
-        # Apply filter-specific priority
-        if ($vars['priorityId'])
-            $form->setAnswer('priority', null, $vars['priorityId']);
-
```

Comment:
Yeah, @protich actually asked the same question on the previous pull request (the one I stuffed up). Here's a copy of my response:

> Because the input is now assigned to $vars['priority'] instead of $vars['priorityId'] it gets automatically loaded into the form (the dynamic field expects the input to be named 'priority'). Hence there is no need to explicitly use $form->setAnswer() to set the priority to $vars['priorityId'], which is what this block of code was doing.  

And here's where you can see my original response in case that is helpful:
https://github.com/torohill/osTicket-1.8/commit/1689ae90302ecfd132cd39a6ba39907ad42c48b1#comments